docs/install-and-run.md

```diff
@@ -3,115 +3,125 @@
 This document shows you how to install EarlGrey and then how to set up and run your first
 test.
 
-## Prerequisites
-
-For EarlGrey to function properly, make sure the app under test has correct launch screen images present for all supported devices (see [iOS Developer Library, Launch Files](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LaunchImages.html)).
-
 ## Install EarlGrey
 
-You can add EarlGrey to Xcode projects in two ways: using CocoaPods, or as a framework. If you wish to contribute to the EarlGrey project, the **EarlGrey.xcodeproj** generated from the [GitHub Installation](#github-installation) section is integrated with the Unit and Functional Test projects for you to work with the source.
+You can add EarlGrey to Xcode projects in three ways: using CocoaPods, Carthage or manually through the Xcode Project.
```

Comment:
>  using CocoaPods, Carthage or manually

Can we add links for each so clicking on Cocoapods takes me directly to cocoapods installation instructions?